cmd/minikube/cmd/config/set_test.go

```diff
@@ -16,11 +16,36 @@ limitations under the License.
 
 package config
 
-import "testing"
+import (
+	"testing"
+	"github.com/minishift/minishift/pkg/minikube/constants"
+	"os"
+)
 
 func TestNotFound(t *testing.T) {
 	err := set("nonexistant", "10")
 	if err == nil {
 		t.Fatalf("Set did not return error for unknown property")
 	}
 }
+
+func TestModifyData(t *testing.T) {
+	constants.ConfigFile = "/tmp/config.json"
```

Comment:
This is not going to work cross platform. You really want to use ioutil.TempDir - https://github.com/minishift/minishift/blob/26a1df15a2d75ee014bce270de5bf620a46d8e4a/pkg/minishift/cache/oc_caching_test.go#L73-L73